WeakAurasOptions/AceGUI-Widgets/AceGUIWidget-WeakAurasDisplayButton.lua

```diff
@@ -1088,37 +1113,97 @@ local methods = {
       end
     else
       -- restore events and layout
-      self.frame:SetScript("OnClick", self.callbacks.OnClickNormal)
-      self.frame:EnableKeyboard(false);
-      self:Enable()
-      if (self.dragging) then
-        self:Drop(true)
+      if self.multiTemp then
+        for _, button in pairs(self.multiTemp.displayButtons) do
+          button.frame:SetScript("OnClick", button.callbacks.OnClickNormal)
+          button.frame:EnableKeyboard(false);
+          button:Enable()
+          if (button.dragging) then
+            button:Drop(true)
+          end
+        end
+        self.multiTemp = nil
+      else
+        self.frame:SetScript("OnClick", self.callbacks.OnClickNormal)
+        self.frame:EnableKeyboard(false);
+        self:Enable()
+        if (self.dragging) then
+          self:Drop(true)
+        end
       end
     end
   end,
   ["ShowTooltip"] = function(self)
   end,
   ["Drag"] = function(self)
-    local uiscale, _, y = UIParent:GetScale(), GetCursorPosition()
```

Comment:
Same here, this is the Drag method of a button. It should not know anything about other buttons.

So the whole logic should be in SetDragging